tests/schemes/test_thrift.py

```diff
@@ -1031,7 +1031,13 @@ def testString(request):
         # values.
         exc = UnexpectedError
     else:
-        exc = ValueExpectedError
+        # If server is using thrift, it will be able to return an invalid
+        # response. thriftrw will fail with a TypeError on invalid values. For
+        # thrift, we'll check manually and raise ValueExpectedError.
+        if use_thriftrw_client:
+            exc = TypeError
```

Comment:
TypeError can also be raised if required arguments in a struct were absent, or on type mismatch. There's no error specific to "return value was absent" raised by thriftrw yet. I can fix that in the next version along with the unexpected exceptions support. Created uber/thriftrw-python#49.

Meanwhile this fix should suffice because this is a very specific corner case that will happen only if the server is using Apache Thrift, the client is using thriftrw, and the server forgets to return a value (which would normally be caught by their tests/manual checks). And when it does occur, there's no good way for the client to handle it except blow up.